### PR TITLE
Fix the font on the home page and migrated the code from test javaSci…

### PR DESCRIPTION
**Pull request by lego134**
…pt to the dev for the FAQ page.

### CODE REVIEW

**Comment by mikelbrierly** [comment]
@lego134 @mikelbrierly

**Comment by lego134** [comment]
@mikelbrierly  @lego134
Changed some code to main.js to understand what I wrote and changed the image on the about me page to what my client liked.

File: HTML/about.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <h1 class="text">About Me</h1>
     <div class="first-about">
         <div class="image">
-            <img src="../image/Teresa-300x271.jpg" width=""  alt="Teresa Raiford smilar photo">
+            <img src= "../image/Teresa .JPG"   alt="Teresa Raiford smilar photo">
```
Comment (mikelbrierly) [inline HTML/about.html:30]: Is there a space before .jpg in the filename?

Comment (lego134) [reply HTML/about.html]: I fix the space in the filename must of prest the space bar.

**Comment by lego134** [comment]
@mikelbrierly @lego134

**Pull request merged**